AndroidSDK/src/com/leanplum/Var.java

```diff
@@ -547,17 +537,7 @@ private int countInternal() {
    *
    * @return A Number value.
```

Comment:
Should we reuse the other javadoc here? `Gets a value from a variable initialized as Number.`